src/txHandler/TransactionHandler.interface.ts

```diff
@@ -45,8 +35,12 @@ export interface ITransactionHandler extends ApprovalHandlerInterface {
     options?: TransactionOptions
   ) => Promise<any>;
 
-  handleWrapEth: (
-    amount: BigNumber,
+  handleWrapEth: (amount: BigNumber, options?: TransactionOptions) => Promise<void>;
+}
+
+export interface IBulkerTransactionHandler extends INotifierManager, ApprovalHandlerInterface {
```

Comment:
The interface is not related to the bulker but to the fact to handle multiple operations. The bulker is an implementation for an EOA. I suggest to rename the interface `IBatchTxHandler` and the function `handleBatchTransactions`. Moreover, it doest have to implement ApprovalHandlerInterface